src/test/groovy/org/prebid/server/functional/tests/pricefloors/PriceFloorsFetchingSpec.groovy

```diff
@@ -1733,6 +1735,33 @@ class PriceFloorsFetchingSpec extends PriceFloorsBaseSpec {
         }
     }
 
+    def "PBS should populate floors enabled = true when floors skip rate 100"() {
+        given: "Default BidRequest with #floorsEnabled and #floorsSkipped"
+        def skipRate = 100
+        def floorsEnabled = true
+        def bidRequest = BidRequest.defaultBidRequest.tap {
+            ext.prebid.floors = new ExtPrebidFloors(enabled: floorsEnabled, skipped: true, skipRate: skipRate)
+        }
+
+        and: "Account with enabled fetch, fetch.url in the DB"
+        def account = getAccountWithEnabledFetch(bidRequest.site.publisher.id)
+        accountDao.save(account)
+
+        when: "PBS processes auction request"
+        floorsPbsService.sendAuctionRequest(bidRequest)
+
+        then: "Bidder request should contain floors data"
+        def bidderRequest = bidder.getBidderRequests(bidRequest.id).last()
+        verifyAll {
+            bidderRequest.ext?.prebid?.floors?.enabled == floorsEnabled
+            bidderRequest.ext?.prebid?.floors?.skipRate == skipRate
+            bidderRequest.ext?.prebid?.floors?.skipped == false
```

Comment:
As far as I understood the initial issue, this should be `true`.

Reply:
The initial issue regarding when we use `skipRate` even if `100` floors enabled should be `true`

Reply:
This looks like a bug then. If floors are enabled and set to be skipped at the rate of 100, meaning that each request will be skipped, so skipped status should be true.